### PR TITLE
Pm 2023 06 issues

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

File: apps/measures/templates/apps_measures/includes/measures_list_item.html [hunk 1]
```diff
@@ -19,8 +19,7 @@ <h3 class="title">{{ content.page_title|truncatechars:102 }}</h3>
             <p class="text">{{ content.contact_organisation_name }}</p>
             <p class="u-b-none">
                 {% for foa in content.get_fields_of_action %}
-                    {{ foa.subset_name }}
-                    {% if not forloop.last %}, {% endif %}
+                    {{ foa.subset_name }}{% if not forloop.last %}, {% endif %}
```
Comment (goapunk) [inline apps/measures/templates/apps_measures/includes/measures_list_item.html:22]: the issue says 
> expected behaviour: Each field of action should be in a separate line
however this just fixes a whitespace in the comma separated list. Is this intentional?

File: digitalstrategie/assets/scss/components/_richtext.scss [hunk 2]
```diff
@@ -18,4 +18,9 @@
     .richtext-image.left {
         max-width: 65%;
     }
+}
+
+// needed for bullet point cut off on safari
+.textile ul > li {
+    margin-left: 1.5px;
```
Comment (goapunk) [inline digitalstrategie/assets/scss/components/_richtext.scss:25]: they are still very slightly cut off for me on safari / browserstack

**Review by goapunk — CHANGES_REQUESTED** [review]
see comments

File: digitalstrategie/assets/scss/components/_richtext.scss (hunk 2 above)
Comment (philli-m) [reply digitalstrategie/assets/scss/components/_richtext.scss:25]: what version

File: apps/measures/templates/apps_measures/includes/measures_list_item.html (hunk 1 above)
Comment (philli-m) [reply apps/measures/templates/apps_measures/includes/measures_list_item.html]: ahh didn't read the issue, thought it was same as before, sorry

**Comment by philli-m** [comment]
@goapunk you got a wip commit here, do you intend to do more?

**Comment by goapunk** [comment]
@philli-m no, I just wasn't sure if that's the right approach. If it is, we can merge the last commit into the first one

**Comment by philli-m** [comment]
@goapunk it looks good so feel free to fixup

**Review by goapunk — APPROVED** [review]
looks  good to me

**Pull request merged**